PhysicsTools/NanoAOD/plugins/JetTaggerTableProducer.cc

```diff
@@ -275,6 +275,7 @@ void JetTaggerTableProducer<T>::produce(edm::Event &iEvent, const edm::EventSetu
                 break;
             }
           }
+          ranked_sv_features.clear();
```

Comment:
I think this change would benefit from a comment stating the `clear()` is a workaround for an `array-bounds` warning/error.

(but don't let this comment delay the build of next 15_0_X release)